shap/plots/_image.py

```diff
@@ -127,3 +132,381 @@ def image(shap_values, pixel_values=None, labels=None, width=20, aspect=0.2, hsp
     cb.outline.set_visible(False)
     if show:
         pl.show()
+
+
+def image_to_text(shap_values):
+
+    from IPython.core.display import display, HTML
```

Comment:
we should put all imports at top, I think that is more standard practice - if this is not required we can have a try/catch around it

Reply:
done